src/applications/pensions/config/chapters/05-financial-information/landMarketable.js

```diff
@@ -0,0 +1,24 @@
+import {
+  yesNoSchema,
+  yesNoUI,
+} from '@department-of-veterans-affairs/platform-forms-system/web-component-patterns';
+
+import { LandMarketableAlert } from '../../../components/FormAlerts';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    'ui:title': 'Income and assets',
+    'ui:description': LandMarketableAlert,
+    landMarketable: yesNoUI({
+      title: 'Is the additional land marketable?',
+      uswds: true,
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      landMarketable: yesNoSchema,
+    },
+  },
+};
```

Comment:
I noticed that the 'LandMarketableAlert' is doing more than being an alert with the included description. We should use composition here to build a description that includes this alert. See the 'LandMarketableDescription' component in the suggested changes:

```suggestion
import React from 'react';
import {
  yesNoSchema,
  yesNoUI,
} from '@department-of-veterans-affairs/platform-forms-system/web-component-patterns';

import { LandMarketableAlert } from '../../../components/FormAlerts';

const LandMarketableDescription = () => (
  <div>
    <p>We want to know if the additional land is marketable.</p>
    <LandMarketableAlert />
  </div>
);

/** @type {PageSchema} */
export default {
  uiSchema: {
    'ui:title': 'Income and assets',
    'ui:description': LandMarketableDescription,
    landMarketable: yesNoUI({
      title: 'Is the additional land marketable?',
      uswds: true,
    }),
  },
  schema: {
    type: 'object',
    properties: {
      landMarketable: yesNoSchema,
    },
  },
};

```